package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": ">",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Jmagero/restaurant-page.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Jmagero/restaurant-page/issues"
+  },
+  "homepage": "https://github.com/Jmagero/restaurant-page#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.25.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "stylelint": "^13.13.0",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.19.0",
+    "webpack": "^5.35.1",
+    "webpack-cli": "^4.6.0"
+  }
+}
```

Comment:
Kindly add `webpack-dev-server` to the `devDependencies` so that the recently added scripts can work as expected. `npm start` will not work if the corresponding dependency is not added (i.e. `webpack-dev-server`).